spec/controllers/api/steps/create_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe Api::StepsController do
+
+  include Devise::TestHelpers
+
+  describe '#create' do
+
+    let(:sequence) { FactoryGirl.create(:sequence) }
+    let(:user) { sequence.user }
+
+    it 'creates a new step sequence' do
+      sign_in user
+      input = { sequence_id: sequence._id.to_s,
+                name: "Scare Birds",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.